.github/review/mcp_trial.py

```
@@ -0,0 +1,345 @@
+def child_ids(events):
+    calls = [event["part"]["state"] for event in events if (event.get("part") or {}).get("tool") == "task"]
+    require(len(calls) == 2, "expected exactly two native child calls")
+    children = {state.get("metadata", {}).get("sessionId"): state.get("input", {}).get("subagent_type") for state in calls}
+    require(len(children) == 2 and None not in children and set(children.values()) == set(ROLES), "invalid independent child identities")
+    return children
+
+
+def validate_sessions(events, exports, model, variant):
+    roots = {event.get("sessionID") for event in events if event.get("sessionID")}
+    require(len(roots) == 1, "ambiguous root session")
+    root = next(iter(roots)); children = child_ids(events); times = []; correctness_high = False
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)